### DISCUSSION TITLE
Nunit3 TestFixtureSource support

### DISCUSSION THREAD

**Discussion by tuscias** [allure-framework/allure2#1756] [Questions & Support]
Hi,
I was wondering if Allure2 for .NET supports Nunit3 TestFixtureSource attribute (https://docs.nunit.org/articles/nunit/writing-tests/attributes/testfixturesource.html)?

Thanks

**Comment by neparij** [comment]
Hi, @tuscias!
TestFixtureSource should work as it designed on NUnit3 side.
Allure Report will contain several test results as on picture below.

Example:
```csharp
[AllureNUnit]
[AllureParentSuite("Root Suite")]
[TestFixtureSource(nameof(FixtureArgs))]
public class AllureTestFixtureSource
{
    private string _word;
    private int _num;
    
    public AllureTestFixtureSource(string word, int num)
    {
        _word = word;
        _num = num;
    }

    static object [] FixtureArgs = {
        new object[] { "Question", 1 },
        new object[] { "Answer", 42 }
    };
    
    [Test]
    public void RunTest()
    {
        Assert.That("Answer", Is.EqualTo(_word));
        Assert.That(42, Is.EqualTo(_num));
    }
}
```

Results:
<img width="992" alt="Screenshot TestFixtureSource" src="https://user-images.githubusercontent.com/17128581/208595560-c2e17a39-d5fb-4fca-8362-13cba07f3ca5.png">

If that doesn't fit your needs - let me know to discuss on it.
